src/sas/system/web.py

```diff
@@ -0,0 +1,24 @@
+class WebLinks:
+    def __init__(self):
+        self.nist_url = "https://www.nist.gov/"
+        self.umd_url = "https://www.umd.edu/"
+        self.sns_url = "https://neutrons.ornl.gov/"
+        self.nsf_url = "https://www.nsf.gov"
+        self.isis_url = "https://www.isis.stfc.ac.uk/"
+        self.ess_url = "https://europeanspallationsource.se/"
+        self.ill_url = "https://www.ill.eu/"
+        self.ansto_url = "https://www.ansto.gov.au/"
+        self.bam_url = "http://www.bam.de/"
+        self.delft_url = "http://www.tudelft.nl/en/tnw/business/facilities/reactor-instituut-delft/"
+        self.inst_url = "https://www.utk.edu"
+        self.diamond_url = "http://www.diamond.ac.uk"
+
+        self.homepage_url = "https://www.sasview.org"
+        self.download_url = 'https://github.com/SasView/sasview/releases'
```

Comment:
This brings users to a list of all releases, including the test builds.

`'https://github.com/SasView/sasview/releases/latest'` would probably be better.

Reply:
I just straight up copied these, but happy to update now.

Reply:
Done